PopulateTools/gobierto
# 1240 - Simpler URL for docs

Closes #1240 

### What does this PR do?

Provides al alternative (and simpler) URL for accessing attachment files.

### How should this be manually tested?

When accessing documents directly, they should be displayed inside an iframe and the URL should match */docs/\<id\>*. Test this:

* With this image: [http://madrid.gobify.net/docs/33](http://madrid.gobify.net/docs/33)
* With this pdf: [http://madrid.gobify.net/docs/87](http://madrid.gobify.net/docs/87)

When rendered inside a CMS page, images should be displayed correctly. Test it in this page: [http://madrid.gobify.net/paginas/pagina-con-archivos](http://madrid.gobify.net/paginas/pagina-con-archivos). Also, when clicking on document link in a page, like this one:

![imagen](https://user-images.githubusercontent.com/9287468/36217959-22bf47c8-11b3-11e8-8bff-53806c965b77.png)

the result URL should match */docs/\<id\>* and the file will be displayed inside an iframe.

When using the "Include attachment" widget in the admin and clicking on "Copy link", a human readable link should be given to the user (*wadus/docs/123*). Test it here: [http://madrid.gobify.net/admin/cms/pages/97/edit?collection_id=13](http://madrid.gobify.net/admin/cms/pages/97/edit?collection_id=13)

### Does this PR changes any configuration file?

No
